toolkit/tools/scheduler/schedulerutils/buildworker.go

```diff
@@ -207,7 +208,7 @@ func parseCheckSection(logFile string) (err error) {
 			}
 			failedLogFile := strings.TrimSuffix(logFile, ".log")
 			failedLogFile = fmt.Sprintf("%s-FAILED_TEST-%d.log", failedLogFile, time.Now().UnixMilli())
-			err = os.Rename(logFile, failedLogFile)
+			err = file.Copy(logFile, failedLogFile)
```

Comment:
I think this is clever, but isn't the logfile still open at the point?  Is the content guaranteed to be flushed to disk before making the copy?

Reply:
The log file is still open but that shouldn't be an issue, since copying it should only read the file.